frontend/libs/studio-components/src/components/StudioCodeFragment/StudioCodeFragment.tsx

```diff
@@ -3,11 +3,14 @@ import React, { forwardRef } from 'react';
 import cn from 'classnames';
 import classes from './StudioCodeFragment.module.css';
 
-export type StudioCodeFragmentProps = HTMLAttributes<HTMLElement>;
+export type StudioCodeFragmentProps = HTMLAttributes<HTMLElement> & {
+  classname?: string;
+};
 
 const StudioCodeFragment = forwardRef<HTMLElement, StudioCodeFragmentProps>(
-  ({ children, className: givenClass, ...rest }, ref) => {
+  ({ children, className: givenClass, classname, ...rest }, ref) => {
```

Comment:
Ref. prev comment, we can also remove this `classname`, since it's already defined as `className: givenClass`. 😊 
```suggestion
  ({ children, className: givenClass, ...rest }, ref) => {
```

Reply:
fixed